src/index.d.ts

```diff
@@ -230,5 +230,17 @@ declare namespace preact {
 	}
 	interface PreactContext<T> extends Context<T> {}
 
-	function createContext<T>(defaultValue: T): Context<T>;
+  function createContext<T>(defaultValue: T): Context<T>;
```

Comment:
The spacing of these lines seems off from the rest of the file (note, I think we use tabs in our src). If there is an editorconfig extension for your editor, using that will just do the right thing for you

Reply:
Ya I think I have mine configured to 2 spaces. I'll fix this.

Reply:
Ok, took a bit to figure out the correct config, but I think I got it setup properly now.